package.json

```diff
@@ -16,8 +16,6 @@
     "@mapbox/unitbezier": "^0.0.0",
     "brfs": "^1.4.0",
     "bubleify": "^0.5.1",
-    "csscolorparser": "^1.0.2",
-    "csscolorparser": "~1.0.2",
```

Comment:
We don't currently install the `dependencies` from `js/style-spec/package.json` so this may still be needed